CountDiv/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace CountDiv;
+
+public static class CountDivClass
+{
+    private static SolutionClass s_sol = new SolutionClass();
```

Comment:
check and apply everywhere
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case:~:text=Use%20camel%20casing%20(%22camelCasing%22)%20when%20naming%20private%20or%20internal%20fields%2C%20and%20prefix%20them%20with%20_.